src/common/test-json.c

```diff
@@ -29,7 +29,7 @@
 static const gchar *test_data =
   "{"
   "   \"string\": \"value\","
-  "   \"number\": 55,"
+  "   \"number\": 55.4,"
```

Comment:
Re-purposing this value sounds like it should have broken the `cockpit_json_get_int (tc->root, "number", 0, &value)` test case; but apparently that succeeds and just truncates the value? It's API by now, so I figure making `_get_int()` fail on non-integers may not be an option, but this still smells error prone.

Also, does that actually truncate or round? Perhaps you can use "55.8" to tell these apart :-)